base/applications/rapps/rappsmsg.mc

```diff
@@ -31,4 +35,7 @@ Removal of the program "%1" is called.
 Language=Polish
 Wywołano usunięcie programu "%1".
 .
+Language=Romanian
+Este apelată ștergerea programului "%1".
```

Comment:
Some other verb besides "a șterge" (which is the closest translation for "to delete") might have fit better. However, this is not a deal-breaker.